patient/README.md

```diff
@@ -0,0 +1,75 @@
+# Patient Mapping
+
+## JSON FHIR > XML HL7
+
+A Patient resource is mapped from a HL7 Patient.
+
+| Mapped to (JSON FHIR Immunization field)        | Mapped from (XML HL7 / other source)                                                              |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| id                                              | set value = Generated GUID                                                                        |
+| meta.profile\[0]                                | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Patient-1"`.         |
+| meta.versionId                                  | fixed value = `1521806400000`                                                                     |
+| indentifier\[0].system                          | fixed value = `https://fhir.nhs.uk/Id/nhs-number`                                                 |
+| indentifier\[0].value                           | `Patient / id [@extension]`                                                                       |
+

```

Comment:
`managingOrganisation` could be mapped here if there is an `AgentOrganisation` in the `AgentDirectory`. It would just be a reference to the mapped organisation, so we could link to the docs for organisation.